config-api/src/main/java/org/keycloak/config/ClusteringOptions.java

```diff
@@ -0,0 +1,54 @@
+package org.keycloak.config;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ClusteringOptions {
+
+    public enum Mechanism {
+        ispn,
+        local
+    }
+
+    public final static Option cache = new OptionBuilder<>("cache", Mechanism.class)
```

Comment:
pls use `static final` everywhere instead of `final static` as to follow the widespread coding convention :)

Reply:
done 👍